src/citrine/gemd_queries/filter.py

```diff
@@ -48,15 +48,15 @@ class AllIntegerFilter(Serializable['AllIntegerFilter'], PropertyFilterType):
 
     Parameters
     ----------
-    lower: str
+    lower: int
         The lower bound on this filter range.
-    upper: str
+    upper: int
         The upper bound on this filter range.
 
     """
 
-    lower = properties.Float('lower')
-    upper = properties.Float('upper')
+    lower = properties.Integer('lower')
+    upper = properties.Integer('upper')
```

Comment:
I was taking a look at the `AllIntergerFilter` case class in the backend, and it looks like it has a field called `inclusive` which defaults to `true`. Was that omission on purpose? I wonder if it is the case that it doesn't add value.

Reply:
I'd missed that.  I'll add it here as well.  🙇